.github/dependabot.yml

```diff
@@ -1,7 +1,91 @@
 version: 2
 updates:
 
-- package-ecosystem: "github-actions"
-  directory: "/"
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
+- package-ecosystem: npm
+  directory: gh-actions/appauth
+  schedule:
+    interval: daily
+  groups:
+    eslint:
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: npm
+  directory: gh-actions/dispatch
+  schedule:
+    interval: daily
+  groups:
+    eslint:
```

Comment:
does this have the effect of batching dependabot prs??

if so - just wow - the coolest thing ive learnt this week!

Reply:
ah i see - new feature!

![image](https://github.com/envoyproxy/toolshed/assets/454682/406ac080-fe65-481c-9840-ec329a0381cd)

ill enable it now

Reply:
Yes ! You could even group every dependencies for each npm project using * pattern